src/components/other/Divider.js

```diff
@@ -30,7 +30,7 @@ const getComputedStyles = ({ isVisible = true, margin } :Props) => {
   `;
 };
 
-const Divider = styled.hr`
+const Divider = styled.div`
```

Comment:
isn't `hr` more semantically appropriate than `div`? in any rare situations where you might need to change the underlying component, you could always use the `as` prop. but honestly the only situation where i could see this happening is if you're using Dividers in between list elements (html5 spec says all elements in a list should be li, including dividers).